internal/service/lakeformation/lf_tag.go

```diff
@@ -185,3 +233,10 @@ func validateLFTagValues() schema.SchemaValidateFunc {
 		validation.StringMatch(regexp.MustCompile(`^([\p{L}\p{Z}\p{N}_.:\*\/=+\-@%]*)$`), ""),
 	)
 }
+
+func Max(x, y int) int {
+	if x > y {
+		return x
+	}
+	return y
+}
```

Comment:
Isn't it defined somewhere else in the provider code?

Reply:
It seems not, only in the tests but not in "internal" code